tilecloud_chain/generate.py

```diff
@@ -107,7 +107,7 @@ def _gene(self, options, gene, dimensions=None):
         elif options.role == 'hash':
             try:
                 z, x, y = (int(v) for v in options.get_hash.split('/'))
-                if 'meta' in gene.layer:
+                if gene.layer.get('meta'):
```

Comment:
Because `meta: off` in the yaml meant there was a 'meta' with a value of False. A presence check meant that layers with meta explicitly set to off were treated as if they had a metatile configuration